PURL-SPECIFICATION.rst

```diff
@@ -343,8 +341,8 @@ To parse a ``purl`` string in its components:
   - Strip the right side from leading and trailing '/'
   - Split this on '/'
   - Discard any empty string segment from that split
-  - Discard any '.' or '..' segment from that split
   - Percent-decode each segment
+  - Discard any '.' or '..' segment from that split
```

Comment:
percent decode first, then discard `..`/`.`.
see the tests in accordance #368

Reply:
LGTM!